components/raftstore/src/store/worker/pd.rs

```diff
@@ -496,6 +507,89 @@ fn hotspot_query_num_report_threshold() -> u64 {
     HOTSPOT_QUERY_RATE_THRESHOLD * 10
 }
 
+struct SlowScore {
```

Comment:
IMO it's better to place a description of the motivation and calculation method of slow score